devops/openshift/db-migrations-job.yml

```diff
@@ -0,0 +1,87 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: ${NAME}
+  annotations:
+    description: "Template for running a job that performs a database migrations"
+labels:
+  project: ${PROJECT}
+  service: ${SERVICE_NAME}
+parameters:
```

Comment:
Is there a benefit to keeping the parameters at the top? I believe that other files have them all at the bottom.